docs/resources/technical-glossary.md

```diff
@@ -0,0 +1,309 @@
+# Technical glossary
+
+A | [B](#b) | [C](#c) | [D](#d) | E | [F](#f) | [G](#g) | [H](#h) | [I](#i) | [J](#j) | K | [L](#l) | [M](#m) | [N](#n) | O | [P](#p) | [Q](#q) | [R](#r) | [S](#s) | [T](#t) | [U](#u) | [V](#v) | [W](#w) | X | Y | Z
+
+---
+
+
+#### B
+
+**Block Explorer**
+
+An application that allows a user to explore the different blocks on a blockchain. It has a record of all the transactions in each block and is often made public to increase blockchain visibility and transparency.  
+
+**Bonded (staking state)**
+
+The non-transferable amount of token staked to the corresponding address and can be unlocked by the `unbond` transaction.
+
+**Byzantine Faults (Double signing)**
+
+A validator is said to make a byzantine fault when they sign conflicting messages/blocks at the same height and round.
+
+
+#### C
+
+**Chain ID**
+A unique identifier for the blockchain. Different prefixes of the Chain ID are used to distinguish between _mainnet_, _devnet_, and _testnet_. 
+For example, the Chain ID of our [testnet](https://crypto.org/docs/getting-started/croeseid-testnet.html) is `testnet-croeseid-2`.
+
+**Chain-abci**
+
+The **A**pplication **B**lock**C**hain **I**nterface connects Tendermint (for consensus operations) to the actual applications.
+
+**Chain-maind**
+
+chain-maind is the all-in-one command-line interface. It supports wallet management, funds transfer, and staking operations. 
+ 
+**Client-rpc**
+
+The JSONRPC interface of the wallet client. It can be used to be integrated with different services and also power the Sample Wallet. It provides nearly the same set of operations as Client-cli does.
+
+**Commission Max Change Rate**
+
+The maximum daily increase of the validator’s commission. This parameter cannot be changed by the validator operator.
+
+**Consensus**
+
+The process of a group of entities to agree on a particular data value, and there are a variety of algorithms used for determining consensus.
+
+**CosmWasm**
+
+It is a new smart contracting platform built for the cosmos ecosystem. It is written as a module that can plug into the Cosmos SDK. 
+
+**Create-validator(Transaction Type: `create-validator`)**
+
+The transaction for joining the network as a validator.
+
+
+#### D
+
+**Delegate (Transaction type: `delegate`)**
+
+The transaction that transfers funds from a transfer address to a staking address for staking purposes. Delegators should perform due diligence on validator candidates before delegating, as well as spreading their stake over multiple validators. It will also help keep the network decentralized.
```

Comment:
```suggestion
The transaction that delagate funds to a validator for staking purposes. Delegators should perform due diligence on validator candidates before delegating, as well as spreading their stake over multiple validators. It will also help keep the network decentralized.
```